content/run.md

```diff
@@ -148,6 +148,101 @@ For dedicated single-cell simulation in order to compare with site-level observa
 
 There are dedicated tools and workflow for running single-cell simulations over Norwegian ecological observation sites using CLM and CLM-FATES, please check [NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform) and follow the instructions there.
 
+## Run a single cell case by creating surface data
```

Comment:
Can you include something about what version of CTSM and python the doc is written for?